Tasks/codepush-promote/task.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id": "4e901d0e-189b-4403-a030-1d06ff8a3b28",
+    "name": "codepush-promote",
+    "friendlyName": "CodePush - Promote",
+    "description": "A VSTS build task for promoting CodePush releases from one deployment to another",
+    "author": "Microsoft Corporation",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "demands": [],
+    "version": {
+        "Major": "1",
+        "Minor": "0",
+        "Patch": "0"
+    },
+    "minimumAgentVersion": "1.83.0",
+    "instanceNameFormat": "Promote $(sourceDeploymentName) to $(targetDeploymentName) for $(appName)",
+    "inputs": [
+        {
+            "name": "authType",
+            "type": "pickList",
+            "label": "Authentication Method",
+            "defaultValue": "ServiceEndpoint",
+            "helpMarkDown": "",
+            "options": {
+                "ServiceEndpoint": "Service Endpoint",
+                "AccessKey": "Access Key"
+            }
+        },
+        {
+            "name": "serviceEndpoint",
+            "type": "connectedService:codepush-auth-key",
+            "label": "Service Endpoint",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The VSTS Service Endpoint that specifies the identity for Code Push.",
```

Comment:
How about "A CodePush service endpoint that is configured with the necessary credentials"?